R/save_to_gama.R

```diff
@@ -50,17 +52,17 @@ generate_obsrate <- function(obsrate, names){
 #' @rdname save_to_gama
 #' @export
 
-save_to_gama <- function(exp, parameter_xml_file) UseMethod("save_to_gama")
+save_to_gama <- function(exp, filename = NULL, path = NULL) UseMethod("save_to_gama")
 
 #' @rdname save_to_gama
 #' @export
-save_to_gama.default <- function(exp, parameter_xml_file)
+save_to_gama.default <- function(exp, filename, path)
                         "Unknown class"
 
 #' @rdname save_to_gama
 #' @export
-save_to_gama.experiment <- function(exp, parameter_xml_file = "") {
-  check_experiment(attr(exp, "experiment"), model(exp))
+save_to_gama.experiment <- function(exp, filename = NULL, path = NULL) {
+  check_experiment(name(exp), model(exp))
```

Comment:
Shouldn't we call the validator here anyway?

Reply:
yes, done